tests/v2/index.test.ts

```diff
@@ -16,22 +16,22 @@ function format(types: string): string {
 describe("cli", () => {
   ["stripe", "manifold", "petstore"].forEach((file) => {
     it(`reads ${file} spec (v2) from file`, () => {
+      const expected = fs.readFileSync(path.join(__dirname, "expected", `${file}.ts`), "utf8");
       execSync(`../../pkg/bin/cli.js specs/${file}.yaml -o generated/${file}.ts`, {
-        cwd: path.resolve(__dirname),
+        cwd: __dirname,
       });
-      expect(fs.readFileSync(path.resolve(__dirname, `expected/${file}.ts`), "utf8")).toBe(
-        fs.readFileSync(path.resolve(__dirname, `generated/${file}.ts`), "utf8")
-      );
+      const generated = fs.readFileSync(path.join(__dirname, "generated", `${file}.ts`), "utf8");
+      expect(generated).toBe(expected);
     });
   });
 
   it("reads swagger.json spec (v2) from remote resource", () => {
-    execSync("../../pkg/bin/cli.js https://api.catalog.stage.manifold.co/swagger.json -o generated/http.ts", {
-      cwd: path.resolve(__dirname),
+    const expected = fs.readFileSync(path.join(__dirname, "expected/http.ts"), "utf8");
+    execSync("../../pkg/bin/cli.js https://petstore.swagger.io/v2/swagger.json -o generated/http.ts", {
```

Comment:
I try not to rely on 3rd party URLs for testing, it won't work when I'm offline and it will fail randomly in case their website is down. It's outside of the scope of this PR, but maybe we could serve the swagger.json file with a local server and then download it from localhost instead?

Reply:
Ha as soon as I put this PR up I just realized: _Duh! Test GitHub with GitHub!_ Now we’re using `raw.githubusercontent.com` URLs (they can be slow, but should at least be reliable?)